OpenKJ/mainwindow.cpp

```diff
@@ -237,7 +225,6 @@ MainWindow::MainWindow(QWidget *parent) :
     _singular->attach();
     delete _singular;
 #endif
-    _singular = new QSharedMemory("SharedMemorySingleInstanceProtectorOpenKJ", this);
```

Comment:
Lines 221 thru 227

`
#ifndef Q_OS_WIN
    // This fixes the program refusing to start and reporting another running instance after a crash on Linux
    // and Mac until you try it twice.
    _singular = new QSharedMemory("SharedMemorySingleInstanceProtectorOpenKJ", this);
    _singular->attach();
    delete _singular;
#endif
`
need to be removed since the original single instance checker was removed.  They reference it and will cause compilation failure on Linux or MacOS.

Reply:
Removed completely. Shouldn't be necessary any more. Only tested on Linux though.